pkg/tests/non-dependant/smcp_install_test.go

```diff
@@ -102,3 +112,17 @@ func assertUninstallDeletesAllResources(t test.TestHelper, ver version.Version)
 				"Still waiting for resources to be deleted from namespace"))
 	})
 }
+
+func getPreviousVersion(t test.TestHelper, ver version.Version) version.Version {
+	var prevVersion *version.Version
+	for _, v := range VERSIONS {
+		if *v == ver {
+			if prevVersion == nil {
+				t.Logf("version %s is the first supported version", ver)
```

Comment:
Why do we need this? This should panic instead of logging & returning nil. Then you don't need the `t` parameter anymore, making the function simpler.

Reply:
Ok

Reply:
Done